docs/download.md

```diff
@@ -72,6 +72,7 @@ New functionality for this release includes:
 * Support for the `concat_ws` operator
 
 Performance improvements for this release include: 
+* Moving RAPIDS Shuffle out of beta
```

Comment:
Sorry, not entirely sure on this one. Should it be under "Performance improvements"?